whisper_server/whisper_api_server.py

```
@@ -99,62 +95,66 @@ def get_optimal_batch_size(device: str) -> int:
     else:
         return 2  # Increased from 1 for better throughput
 
+
 def validate_model_name(model: str) -> tuple[bool, Optional[str]]:
```

Ruff (0.13.1)

Redefinition of unused `validate_model_name` from line 24

(F811)